core/src/main/scala/org/apache/spark/scheduler/BlacklistTracker.scala

```diff
@@ -336,9 +336,9 @@ private[scheduler] object BlacklistTracker extends Logging {
    */
   def isBlacklistEnabled(conf: SparkConf): Boolean = {
     conf.get(config.BLACKLIST_ENABLED) match {
-      case Some(enabled) =>
+      case enabled =>
         enabled
-      case None =>
+      case _ =>
```

Comment:
I see, @guoxiaolongzte I think this code is written this way on purpose and we can't change this behavior. Removing the possibility of no setting does change semantics and it's not worth it. CC @squito